src/algorithms/GenericFunctions.jl

```diff
@@ -0,0 +1,31 @@
+###############################################################################
+#
+#   GenericFunctions.jl : Functions for Generic types
+#
+###############################################################################
+
+# TODO: Move from CRT from Hecke to AbstractAlgebra?
+# Currently no implementation, only example on how the arbitrary inputs `crt`
+# should look like.
+@doc Markdown.doc"""
+    crt(r::Array{T, 1}, m::Array{T, 1}) where T
```

Comment:
Should this perhaps be
```suggestion
    crt(r::AbstractVector{T}, m::AbstractVector{T}) where T
```

Reply:
Thank you!